### PR TITLE
Added Global Enable Setting

### PR DESCRIPTION
**Pull request by dieskim**
I have added and edited System Settings as per https://developer.piwik.org/guides/plugin-settings to add a Global Enable Setting. This will ALWAYS require every user to add a Two Factor code when logging in. If the user does not have two factor set they will not be able to log in. This should be enabled after the admin has gotten all users to add two factor - it can also be disabled for a user to login and set it up and then enabled again

CAUTION: Set Up Two factor on your global user before you Enable this feature or you wont be able to log in

### CODE REVIEW

**Comment by dieskim** [comment]
Another thing was that you should be able to disable that setting via config/config.ini.php 

```
[GoogleAuthenticator]
globalEnable = false

```
but somehow that did not work for me in testing

as per https://developer.piwik.org/guides/plugin-settings

File: SystemSettings.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/**
```
Comment (sgiehl) [inline SystemSettings.php:2]: guess here is a `<?php` missing

Comment (dieskim) [reply SystemSettings.php:2]: Fixed sorry

**Review by dieskim — COMMENTED** [review]
Sorry fixed

**Comment by sgiehl** [comment]
Closing this one, as 2FA will be integrated in core in the future. This feature will be available then.

**Pull request closed without merging**